core/eolearn/core/eodata_io.py

```diff
@@ -219,6 +220,17 @@ def walk_main_folder(filesystem: FS, folder_path: str) -> Iterator[Tuple[Feature
         else:
             ftype_str, fname = raw_path, ...
 
+        if ftype_str == "timestamp":
+            warnings.warn(
+                (
+                    f"EOPatch at {filesystem.getsyspath(folder_path)} contains the deprecated `timestamp` feature. "
+                    "Save the EOPatch again to update it."
```

Comment:
```suggestion
                    f"EOPatch at {filesystem.getsyspath(folder_path)} contains the deprecated `timestamp` feature."
                    " The old name will no longer be valid in the future. You can re-save the `EOPatch` to update it."
```

Forgot it also affects files. This could be a bit annoying for a while :)